libmscore/note.cpp

```diff
@@ -246,6 +246,16 @@ Note::Note(const Note& n, bool link)
       else
             _tieFor = 0;
       _tieBack  = 0;
+      // clone forward-spanners, leaving end note empty; at a later time,
+      // Score::connectTies() will hopefully find the correct end note
+      for (Spanner* oldSp : n._spannerFor) {
+            Spanner* newSp = static_cast<Spanner*>(oldSp->clone());
```

Comment:
So when adding a linked staff to an existing one, we will clone the chord, so the notes, and so the spannerFor.
Here https://github.com/musescore/MuseScore/blob/master/libmscore/excerpt.cpp#L710 we already clone spanner while cloning the staff. Don't we create too much of these clones ? Maybe we should centralized the cloning here, on the back spanners? and do the linking later on?